components/build-templates/base/e2e/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: appstudio-pipeline
+secrets:
+  - name: quay-push-secret
+imagePullSecrets:
+  - name: quay-push-secret
```

Comment:
Does this cause any issues in the dev environment? quay-push-secret is only in stage and prod https://github.com/redhat-appstudio/infra-deployments/pull/2800